lib/v2/screens/transfer/send/send_confirmation/components/custom_transaction_success_dialog.dart

```diff
@@ -0,0 +1,166 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:intl/intl.dart';
+import 'package:seeds/v2/datasource/remote/model/custom_transaction_model.dart';
+import 'package:seeds/v2/design/app_theme.dart';
+import 'package:seeds/v2/components/custom_dialog.dart';
+import 'package:seeds/v2/components/profile_avatar.dart';
+import 'package:seeds/v2/constants/app_colors.dart';
+import 'package:seeds/v2/i18n/transfer/transfer.i18n.dart';
+
+class CustomTransactionSuccessDialog extends StatelessWidget {
```

Comment:
we have a dialog class we extend from when creating dialogs. 

Is this dialog different? Should this extend from our base dialogs?

Reply:
I did not change this at all - I just made a second one based on TransactionSuccessDialog

I didn't want to change inheritance, or anything about it, since I did not want to break it.